test/all/index.js

```diff
@@ -159,6 +164,31 @@ test('createFile', function(t) {
   });
 });
 
+test('upload', function(t) {
```

Comment:
Since we now have a symetric `download`/`upload`, can we move the `download` test down below `upload` and put `upload` after the `delete`?

So essentially the flow will be:
* Upload file using `createFile`
* Download file using `getFile`
* Delete file
* Upload file using `upload`
* Download file using `download`
* Delete bucket